qfieldsync/utils/file_utils.py

```diff
@@ -104,3 +105,22 @@ def slugify(text: str) -> str:
     slug = re.sub(r'[-]+', '-', slug)
     slug = slug.lower()
     return slug
+
+
+def copy_images(source_folder, destination_folder):
+    if os.path.isdir(source_folder):
+        if not os.path.isdir(destination_folder):
+            os.mkdir(destination_folder)
```

Comment:
Can we always assume the parent folder exists in every case?
I guess yes, but if not https://stackoverflow.com/a/600612/2319028

Reply:
In the context, yes. Since we only call this, when there is a project-folder.